client/src/App.css

```diff
@@ -9,7 +9,7 @@
 
 @media (prefers-reduced-motion: no-preference) {
   .App-logo {
-    animation: App-logo-spin infinite 20s linear;
+    animation: App-logo-spin infinite 1s linear;
```

Comment:
Was just playing around and see what I could change on the default application. I didn't remove it all because I didn't know if it would mess something up or not.